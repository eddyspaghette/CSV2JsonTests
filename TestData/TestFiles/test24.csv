ッ!あ!が!ざ!℁!⏳
ツ!ぃ!ぎ!せ!℟!▓
ヅ!い!け!₠!!○
ミ!ぅ!!₡!ℬ!☝
デ!う!こ!₢!Δ!⭐
ト!!ご!₣!!😵
!え!さ!℀!ↂ!🍕
ッ!あ!が!ざ!℁!⏳
デ!う!!₢!Δ!⭐
