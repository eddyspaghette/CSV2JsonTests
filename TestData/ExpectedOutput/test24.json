[
 {
   "FIELD1": "ッ!あ!が!ざ!℁!⏳"
 },
 {
   "FIELD1": "ツ!ぃ!ぎ!せ!℟!▓"
 },
 {
   "FIELD1": "ヅ!い!け!₠!!○"
 },
 {
   "FIELD1": "ミ!ぅ!!₡!ℬ!☝"
 },
 {
   "FIELD1": "デ!う!こ!₢!Δ!⭐"
 },
 {
   "FIELD1": "ト!!ご!₣!!😵"
 },
 {
   "FIELD1": "!え!さ!℀!ↂ!🍕"
 },
 {
   "FIELD1": "ッ!あ!が!ざ!℁!⏳"
 },
 {
   "FIELD1": "デ!う!!₢!Δ!⭐"
 }
]